### PR TITLE
LPS-181622 A new user with full Administrator permissions do not select roles

### PR DESCRIPTION
**Pull request by alvarolealdl**

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/Inner/CaseResult/index.tsx [hunk 1]
```diff
@@ -57,6 +57,9 @@ const CaseResult = () => {
 		[caseResult.attachments]
 	);
 
+	const buttonMustBeVisible =
+		Object.keys(caseResult.actions).length >= 3 ?? false;
+
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/Inner/CaseResult/index.tsx:62]: You just need to look for `edit` permission not the rest

```suggestion
	const hasCaseResultEditPermission =
		!!caseResult.actions?.edit
```

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/Inner/CaseResult/index.tsx [hunk 2]
```diff
@@ -232,6 +235,7 @@ const CaseResult = () => {
 											}
 										/>
 									),
+									visible: buttonMustBeVisible,
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/Inner/CaseResult/index.tsx:238]:
```suggestion
									visible: hasCaseResultEditPermission,
```

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/Inner/CaseResult/CaseResultHeaderActions.tsx [hunk 3]
```diff
@@ -62,11 +62,18 @@ const CaseResultHeaderActions: React.FC<{
...
+		Object.keys(caseResult.actions).length >= 3 ?? false;
+
 	return (
 		<>
 			<AssignModal modal={modal} />
 
-			<ClayButton.Group className="mb-3 ml-3" spaced>
+			<ClayButton.Group
+				className="mb-3 ml-3"
+				hidden={!buttonMustBeVisible}
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/Inner/CaseResult/CaseResultHeaderActions.tsx:74]:
```suggestion
				hidden={! hasCaseResultEditPermission}
```

**Review by kevenleone — CHANGES_REQUESTED** [review]
Thanks @alvarolealdl 

take a look

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/Inner/CaseResult/index.tsx (hunk 1 above)
Comment (Elvinho) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/Inner/CaseResult/index.tsx:62]: there is no edit permission, but it contains updateBatch is that right?

**Pull request closed without merging**